doc/programming_model/attributes_quantization.md

```diff
@@ -492,22 +517,3 @@ impossible to apply them on-the-fly without potential accuracy loss.
            attr);   // the attributes describe the quantization flow
 // ...
 ~~~
```

Comment:
do you plane to also extent either doc or examples (I think it would be better!) with MXFP8 example?